sql/core/src/main/scala-2.13/org/apache/spark/sql/execution/streaming/StreamProgress.scala

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Should we leave a note like `//  Note: this class supports Scala 2.12. A parallel source tree has a 2.13 implementation.` in the moved `StreamProgress.scala`?